packages/pages/src/dev/server/autoInit.ts

```diff
@@ -45,10 +48,21 @@ Let's connect to your Yext Account
         console.log(error.message);
         process.exit(1);
       });
-    fs.writeFileSync(
-      ".yextrc",
-      `accountId: ${accountId}\nuniverse: ${universe}`
-    );
+    if (scope) {
+      fs.appendFileSync(
+        ".yextrc",
+        `"${scope}":\n` +
```

Comment:
alternatively you could also use a JSON object then parse to string instead of writing in the colons and \n characters

Reply:
I could! I could just leave it as json actually (i think json is a valid format in yaml), only thing I can't add is the comment I think